backend/migrator/migration/prod/1.16/0002##delete_email_duplicated_principal.sql

```diff
@@ -22,6 +22,8 @@ LOOP
   SELECT * INTO first_user FROM principal WHERE email = row_data.email ORDER BY id LIMIT 1;
   UPDATE setting SET creator_id = first_user.id WHERE creator_id = row_data.id;
   UPDATE setting SET updater_id = first_user.id WHERE updater_id = row_data.id;
+  UPDATE role SET creator_id = first_user.id WHERE creator_id = row_data.id;
```

Comment:
https://github.com/bytebase/bytebase/blob/main/backend/migrator/migration/prod/1.17/0000%23%23custom_role.sql

We don't have role till 1.17. remove this section.